### DISCUSSION TITLE
SEB 3.3.2 issue on macOS 10.13.6

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-mac#378] [Q&A]
Hi,

I am trying to launch Safe Exam Browser on my macOS 10.13.6. However, I am being presented with this error.

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/73a2a582-5eac-4aaa-aa3b-9f8c57bb7fb0)

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/13cb4f71-2d5b-43bb-aba1-06a143ff0f98)

It is stated that 10.13 is supported. However, it does not seem to be the case. Can this please be confirmed?

**Comment by dbuechel** [comment]
@fkm97 Please always open issues of SEB for Windows or macOS in the respective repository. The seb-win-refactoring repository is for Windows issues only.
